tools/clang/test/CodeGenSPIRV/shader.debug.runtimearray.hlsl

```diff
@@ -0,0 +1,18 @@
+// Run: %dxc -T ps_6_0 -E main -fspv-debug=vulkan
```

Comment:
Please use `// RUN:` (upper cases) instead of "// Run:`.

Reply:
Oh yes. Thanks for the reminder! I was copying from an old version of the tree!